src/App.js

```diff
@@ -1,4 +1,4 @@
-import Radio from './Components/Radio'
+import Radio from './components/Radio'
```

Comment:
You also have to change the casing of the "components" folder under the "src" folder, then only this line will work.

Reply:
I have done that

Reply:
<img width="1440" alt="Screenshot 2021-10-05 at 1 42 36 AM" src="https://user-images.githubusercontent.com/52633729/135918468-ef7a607b-0a9e-4115-8632-18af0954e095.png">


I can't see any changes being made here.

Reply:
Git wasn't detecting rename when I renamed directly through vs code but It was detected when I renamed the folder from terminal